lib/nostrum/struct/message.ex

```diff
@@ -70,12 +81,25 @@ defmodule Nostrum.Struct.Message do
   @typedoc "List of attached files in the message"
   @type attachments :: [Attachment.t()]
 
-  @typedoc "List of embedded content in the message"
-  @type embeds :: [Embed.t()]
+  @typedoc """
+  List of Message Components
+  """
+  @type components :: [Component.t()]
 
   @typedoc """
-  Reactions to the message.
+  Array of Message Sticker Item Objects
   """
+  @type sticker_items :: [Sticker.t()]
+
+  @typedoc """
+  Message interaction object.
```

Comment:
This should not have a trailing period to match the style of the other docstrings:

```suggestion
  Message interaction object
```